style.css

```diff
@@ -1 +1,110 @@
-/*  Add your styling here */
+body{
+  display: flex;
+  flex-direction: column;
+  background-color: darkred;
+}
+*{ 
+    box-sizing:border-box;
+    font-family:"Dosis", sans-serif; 
+  }
+header{
+   display: flex;
+   flex-direction: column;
+   }
+  .menu{
```

Comment:
Make sure to use correct indentation in your HTML and CSS. Whitespace makes all the difference when reading someone else's code!